core/arch/riscv64/riscv64.asm

```diff
@@ -264,7 +264,16 @@ GLOBAL_LABEL(_dynamorio_runtime_resolve:)
  */
         DECLARE_FUNC(dynamorio_clone)
 GLOBAL_LABEL(dynamorio_clone:)
-/* FIXME i#3544: Not implemented */
+        sd      ARG1, 8 (ARG2)
+        sd      ARG6, 0 (ARG2) /* Func is now on TOS of newsp. */
```

Comment:
This is not right, the stack is growing the other way around.

Reply:
I forgot to increase the stack size. Sorry.

Reply:
`Newsp` -> `newsp`, `Func` -> `func`, these are variable names, it's better NOT be capitalized IMO.

Reply:
Done.